docs/references/skyh5_memo.tex

```diff
@@ -0,0 +1,221 @@
+	\item \textbf{representation\_type}: \textit{string} The representation type, one of ``spherical'', ``cartesian'' or ``cylindrical''
+	\item Representation component names (e.g. \textbf{ra}, \textbf{dec}, \textbf{alt}, \textbf{az}): \textit{float} Two or three such components must be present, 	which ones are required depend on the frame. These are one-dimensional arrays of size (Ncomponents).
+	\end{itemize}
+And may include any other attributes accepted as input parameters for an \texttt{astropy SkyCoord} object (e.g. obstime, equinox, location).
+Each of these datasets may have ``unit'' and ``object\_type'' attributes and may be either a scalar or a one-dimensional array of size (Ncomponents) as appropriate.
+(\textit{skycoord})
+
+\item \textbf{nside}: \textit{int}
+The HEALPix nside parameter. Note this is \textbf{required} if the component\_type is ``healpix'' and should not be defined otherwise. (\textit{nside})
+
+\item \textbf{hpx\_order}: \textit{string}
+The HEALPix pixel ordering convention, either ``ring'' or ``nested''.
+Note this is \textbf{required} if the component\_type is ``healpix'' and should not be defined otherwise. (\textit{hpx\_order})
+
+\item \textbf{hpx\_frame}:  A nested dataset that contains the information to describe an \texttt{astropy} coordinate frame giving the HEALPix coordinate frame.
+This is similar to the skycoord dataset described above but it does not contain the representation\_type or the representation component names.
+Note this is \textbf{required} if the component\_type is ``healpix'' and should not be defined otherwise.
+The keys must include:
+	\begin{itemize}
+	\item \textbf{frame}: \textit{string} The name of the coordinate frame (e.g. ``icrs'', ``fk5'', ``galactic''). Must be a frame supported by \texttt{astropy}.
+	\end{itemize}
+And may include any other scalar attributes accepted as input parameters for an \texttt{astropy SkyCoord} object (e.g. obstime, equinox, location).
+Each of these datasets may have ``unit'' and ``object\_type'' attributes as appropriate.
+(\textit{hpx\_frame})
+
+\item \textbf{hpx\_inds}: \textit{int}
+The HEALPix indices for the included components. Does not need to include all the HEALPix pixels in the map.
+This is a one-dimensional array of size (Ncomponents).
+Note this is \textbf{required} if the component\_type is ``healpix'' and should not be defined otherwise. (\textit{hpx\_inds})
+
+\item \textbf{freq\_array}: \textit{float}
+Frequency array giving the nominal (or central) frequency in a unit that can be converted to Hz.
+Note this is \textbf{required} if the spectral\_type is ``full'' or ``subband'' and should not be defined otherwise. (\textit{freq\_array})
+
+\item \textbf{freq\_edge\_array}: \textit{float}
+Array giving the frequency band edges in a unit that can be converted to Hz, only required if spectral\_type is ``subband''.
```

Comment:
You say that freq_edge_array is only required for subband types twice in this description.